puppet/modules/quickstack/manifests/pacemaker/redis.pp

```diff
@@ -0,0 +1,14 @@
+class quickstack::pacemaker::redis(
+  $bind_host             = '127.0.0.1',
```

Comment:
How will using 127.0.0.1 work in an HA setup?  if we are not load blancing redis (similar to memcached), it seems like we would want to bind on either 0s or the map_params("local_bind_addr") value, otherwise each node would only get the cache from there instead of potentially elsewhere, if that is where the right copy is?